examples/jsm/nodes/accessors/InstanceNode.js

```diff
@@ -95,13 +95,13 @@ class InstanceNode extends Node {
 
 	update( /*frame*/ ) {
 
-		if ( this.instanceMesh.instanceMatrix.version !== this.buffer.version ) {
+		if ( this.instanceMesh.instanceMatrix.usage !== DynamicDrawUsage && this.instanceMesh.instanceMatrix.version !== this.buffer.version ) {
```

Comment:
In `Attribute.js` the buffer is updated if the version is different _or_ the usage is `DynamicDrawUsage `. Do you mind explaining why the check is here differently? Shouldn't it be:
```js
if ( this.instanceMesh.instanceMatrix.usage === DynamicDrawUsage || this.instanceMesh.instanceMatrix.version !== this.buffer.version ) {
```

Reply:
The first condition bufferAttribute.usage === DynamicDrawUsage in Attributes.js is sufficient for InstanceNode to work if the usage of instanceMatrix is initialized with DynamicDrawUsage. This is because the usage is forwarded when the node is setup:
```js
const bufferFn = instanceAttribute.usage === DynamicDrawUsage ? instancedDynamicBufferAttribute : instancedBufferAttribute;
```

This means that forwarding the version in the update() method to the isolated interleavedBuffer is not necessary for the node to work when the usage is dynamic. That's why the condition uses this.instanceMesh.instanceMatrix.usage !== DynamicDrawUsage to forward the version only if the attribute is "static".

Also this code is necessary because the interleavedBuffer used in `InstanceNode` is isolated inside the node, and so bumping the version of `this.instanceMesh.instanceMatrix` manually with `needsUpdate = true` has no effect.